### PR TITLE
Testing using jest

### PR DESCRIPTION
**Pull request by Georjane**
In this PR, we created several tests for our functions to test both for positive and negative scenarios. You can run these test using `npm test`

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've both done quite well here.**👍

- However, please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: _tests_/categoryClass.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const category = require('../src/components/categoryClass');
```
Comment (mikenath223) [inline _tests_/categoryClass.test.js]: - [ ] Please convert this to es6 import syntax. For this to work well you would need to use es6 export syntax for the class module. So you can simply convert `module.export = {Category}` to `export default Category`.

File: _tests_/todoClass.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const todo = require('../src/components/todoClass');
```
Comment (mikenath223) [inline _tests_/todoClass.test.js]: - [ ] Kindly make similar changes here as stated above.

**Review by JamesOkunlade — APPROVED** [review]
**Hi guys!! 🙂** 

Good job implementing the changes requested by the previous reviewer.

Your project now meets the minimum requirement to pass. It's a thumb up 👍 

Go ahead and merge.

## Approved 👍

**Pull request merged**